pokedex/lib/main.dart

```diff
@@ -0,0 +1,273 @@
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert' as convert;
+import 'package:pokedex/models/Pokemon.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+List<Pokemon> pokemon_list = [];
+List<String> pokemonImage_list = [];
+ScrollController _scrollController = ScrollController();
+int offset = 20;
+
+class MyApp extends StatefulWidget {
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+
+  bool isLoading = true;
+
+  @override
+  void initState() {
+    getPokemon(offset);
+
+    Future.delayed(const Duration(milliseconds: 5000), () {
```

Comment:
why do you need to delay it for 5 seconds?

Reply:
to load in the first 20 batch of pokemon first before build

Reply:
does it really need to?